lib/std/Build/Step/Compile.zig

```diff
@@ -64,6 +64,8 @@ initial_memory: ?u64 = null,
 max_memory: ?u64 = null,
 shared_memory: bool = false,
 global_base: ?u64 = null,
+/// Tells the linker to not output an entry point.
+no_entry: ?bool = null,
```

Comment:
This probably wasn't meant to be `bool`?

Reply:
The reasoning behind `?bool` was we only emit the flag `-fentry` or `-fno-entry` when this field is set. When left `null`, we simply provide neither flag to the CLI, similar to a user being not required to provide either flag to build-{exe|lib}.

Reply:
I didn't notice the separate `entry_symbol_name` field.